website: Extend the explanation in the iperf3 tutorial (#620)
clusterlink-net/clusterlink: website/static/files/tutorials/deploy-clusterlink.md

    clusterlink create peer-cert --name server
    ```

>     All peer certificates (client and server) should be created from the same Fabric CA files.

Reviewer: ```suggestion
    All peer certificates (i.e., for the `client` and `server` clusters, in this tutorial) should be created from the same fabric CA files.
```

Reviewer: Since fabric is lower case in line 2, I changed to `fabric` here too.

>     In this tutorial, we assume the server has access to the Fabric certificate created in the `default_fabric` folder.
>     In case the server doesn't have access, the Fabric certificate should be copied from the client to the server.

Reviewer: ```suggestion
    In this tutorial, we assume the `server` cluster creation has access to the fabric certificate stored in the
     `default_fabric` folder. If it doesn't, the fabric certificate should be copied from the `client` to the `server`.
```
